### DISCUSSION TITLE
`knownAliases` and `knownAliasesLower` no longer returned

### DISCUSSION THREAD

**Discussion by mdashlw** [HypixelDev/PublicAPI#561] [General]
`knownAliases` and `knownAliasesLower` don't exist anymore, is this a permanent change? will there be alternatives?

**Comment by Picsou993** [comment]
With the removal of the Username History on the Mojang API we can consider that this is a permanent change and that no alternatives will be provided: https://help.minecraft.net/hc/en-us/articles/8969841895693
